packages/fedify/src/runtime/langstr.ts

```diff
@@ -1,36 +1,34 @@
-import { type LanguageTag, parseLanguageTag } from "@phensley/language-tag";
-
 /**
  * A language-tagged string which corresponds to the `rdf:langString` type.
  */
 export class LanguageString extends String {
-  readonly language: LanguageTag;
+  readonly locale: Intl.Locale;
```

Comment:
I do like this renaming, but we should write an changelog entry about this change.

Also, we need a JSDoc comment here!